installers/macos-pkg-setup-template.sh

```diff
@@ -0,0 +1,67 @@
+set -e
+
+PYTHON_FULL_VERSION="{{__VERSION_FULL__}}"
+PYTHON_PKG_NAME="{{__PKG_NAME__}}"
+MAJOR_VERSION=$(echo $PYTHON_FULL_VERSION | cut -d '.' -f 1)
+MINOR_VERSION=$(echo $PYTHON_FULL_VERSION | cut -d '.' -f 2)
+
+PYTHON_MAJOR=python$MAJOR_VERSION
+PYTHON_MAJOR_DOT_MINOR=python$MAJOR_VERSION.$MINOR_VERSION
+PYTHON_MAJORMINOR=python$MAJOR_VERSION$MINOR_VERSION
```

Comment:
Maybe rename the variable to PYTHON_MAJOR_MINOR just to be more readable :)

Reply:
I fully support this suggestion